pkg/cli/bool_flag.go

```diff
@@ -6,6 +6,8 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+var _ Flag = new(BoolFlag)
```

Comment:
We usually don't comment on this, since it is always done for one purpose only, namely to ensure that the given type implements all the interface methods. But why now.

https://github.com/search?q=repo%3Ahashicorp%2Fterraform%20var%20_%20&type=code